lib/tasks/data.rake

```diff
@@ -1,41 +1,9 @@
 DATA_MIGRATION_SERVICES = [
   # do not delete or edit this line - services added below by generator
-  'DataMigrations::BackfillCandidatesSubmissionBlocked',
-  'DataMigrations::DropBlockFraudulentSubmissionFeatureFlag',
-  'DataMigrations::BackfillMonthlyStatisticsData',
-  'DataMigrations::FixCandidateAPIUpdatedAt',
-  'DataMigrations::BackfillCarriedOverApplicationsDegreesComplete',
-  'DataMigrations::BackfillWorkHistoryStatusForCurrentCycle',
-  'DataMigrations::BackfillWorkHistoryStatus',
-  'DataMigrations::BackfillNotCompletedExplanation',
-  'DataMigrations::BackfillCoursesForNextCycle',
-  'DataMigrations::BackfillRestructuredImmigrationStatus',
-  'DataMigrations::BackfillInvalidProviderRelationshipPermissions',
-  'DataMigrations::PopulateApplicationChoiceCurrentRecruitmentCycleYear',
-  'DataMigrations::PopulateApplicationChoiceProviderIds',
   'DataMigrations::RemoveDuplicateProvider',
-  'DataMigrations::BackfillReferencesCompleted',
-  'DataMigrations::CleanseEocChasersSentData',
-  'DataMigrations::BackfillSetupInterviewsPermission',
-  'DataMigrations::FixEmptyOfferConditions',
-  'DataMigrations::RemoveSurplusReferenceSelections',
-  'DataMigrations::BackfillCandidateAPIUpdatedAt',
   'DataMigrations::BackfillNoneHesaDisabilitiesCodes',
```

Comment:
Could we remove `BackfillNoneHesaDisabilitiesCodes` and `DeleteUuidlessCourses`, and keep the last two as those have def been run?